shared/test/render-dumb-sheet.js

```diff
@@ -9,44 +9,62 @@ import DumbSheetItem from '../dev/dumb-sheet/item'
 
 const PADDING = 25
 
-ipcRenderer.on('display', (ev, msg) => {
-  const map = dumbComponentMap[msg.key]
-  const mockKey = msg.mockKey
+window.load = (options) => {
```

Comment:
the diff is more complicated than it really is. Mostly due to wrapping this stuff in window.load